index.ts

```diff
@@ -129,6 +130,7 @@ export {
   nodes,
   tags,
   functions,
+  attributes,
```

Comment:
what is the use case for exposing  this?

```
export const globalAttributes: AttributesSchema = {
  class: { type: Class, render: true },
  id: { type: String, render: true },
};
```

Reply:
Just to align it with the other primitive collections: `nodes`, `tags`, `functions`. Do you think we should call it `globalAttributes` instead?